docs/node-mixin/config.libsonnet

```diff
@@ -52,6 +52,7 @@
     fsSpaceAvailableCriticalThreshold: 5,
     fsSpaceAvailableWarningThreshold: 3,
 
-    grafana_prefix: '',
+    dashboardNamePrefix: 'Node Exporter / ',
+    dashboardTags: ['node-exporter-mixin'],
```

Comment:
WDYT about doing 2 tags (node-exporter and mixin) instead of one? The mixin tag would fit nicely with the same tag set in kubernetes-mixin dashboards.